### ISSUE TITLE
No Images shown in /public/ with docker install

### ISSUE THREAD

**Issue by Blackraz0r** [XGProyect/XG-Proyect-v3.x.x#444]
**Describe the bug**
I have no Images when i open dockerhostIP:httpport:
https://i.imgur.com/8Yw0PG7.jpg
the admin interface is working fine:
https://i.imgur.com/XKFDj81.jpg

**To Reproduce**
1.
enter docker host and pull git (ubuntu 20.04)
```mkdir /opt/XGP
cd /opt/XGP
sudo git clone https://github.com/XGProyect/XG-Proyect-v3.x.x.git
cd XG-Proyect-v3.x.x.git
```

2. i then altered the compose file to work with networks, this is my compose file:
```
version: "3.8"
services:
  xgproyect-v3:
    build:
      context: .
      dockerfile: Dockerfile
    container_name: xgproyect-v3
    ports:
      - '8010:80'
      - '8011:443'
    volumes:
      - ./src/upload:/var/www/html
    depends_on:
      - db
    networks:
      - xgp_network
  db:
    image: mysql:5.7
    ports:
      - 127.0.0.1:33060:3306 # change ip if required
    volumes:
      - ./data:/docker-entrypoint-initdb.d
    environment:
      MYSQL_DATABASE: xgp
      MYSQL_ROOT_PASSWORD: root
    networks:
      - xgp_network
  phpmyadmin:
    image: phpmyadmin/phpmyadmin
    ports:
        - '8012:80'
    environment:
        MYSQL_ROOT_PASSWORD: root
        PMA_HOST: db
    networks:
        - xgp_network

networks:
    xgp_network:
```

3. Start the Stack

```
sudo docker-compose up -d
```

4. give write acces for config file as the instalation site (ip/install) told me
chmod -r 777 /location
installation then went fine.
after install i can access the site but there are no images. just plain text.

when i access via (ip of docker host and port)
http://10.250.30.220:8010/public/
i see the folowing:
https://i.imgur.com/8Yw0PG7.jpg
(link from above)
when i access via 
http://10.250.30.220:8010/admin.php?page=home
i can see this:
https://i.imgur.com/VnaMGIa.jpg
(image from admin interface working)

**Expected behavior**
i have images in the game interface instead of only text
**Screenshots**

**Desktop (please complete the following information):**
 - OS: Win10/Ubuntu2004
 - Browser Mozilla/Chromium

 
 **Server (please complete the following information):**
 - the versions provided by the latest docker-compose script

**Comment by Blackraz0r** [comment]
the ingame interface looks like this:
https://i.imgur.com/VnaMGIa.jpg

**Comment by duhow** [comment]
Reproducible with commit 5d02f9e0b32cd657f0529323a88281a796690660 v3.2.0 .

Note that homepage (login) is loading from `/public`, but you should serve it directly from root `/` .
Otherwise content loads from `http://host:8010/public/public/css/home/reset.css` and returns invalid content.

**Comment by duhow** [comment]
Suggesting to modify the `.htaccess` file to redirect non-existing files from `/public` to previous directory?
Will try to provide a fix.

**Comment by LucasKovacs** [comment]
Resolved on 3.3
